docs/source/faqs.rst

```diff
@@ -110,6 +110,22 @@ Model training
     `here <https://lightning-pose.readthedocs.io/en/latest/source/user_guide/config_file.html>`_
     (this link takes you to another set of docs specifically for Lightning Pose).
 
+.. dropdown:: What is and how many epochs should I use for training?
+    **What is an epoch?**
```

Comment:
add empty line between `..dropdown` and `**What is an epoch**` (otherwise the dropdown doesn't get rendered properly, just an rst file quirk)

Reply:
also, add a tag that we can refer to later if necessary, called `.. _faq_epochs` - like the previous comment, there needs to be an empty line above and below the tag